plugins/asus-hid/fu-asus-hid-device.c

```diff
@@ -87,6 +96,11 @@ fu_asus_hid_device_probe(FuDevice *device, GError **error)
 		fu_device_add_child(device, dev_tmp);
 	}
 
+	g_signal_connect(device,
```

Comment:
i think this wants to go into _init really -- e.g. we can do probe more than once in some circumstances and then you'll get duplicate events. I also think you want to disconnect this (ideally with g_signal_handler_disconnect) so there are no issues in the self tests.

Reply:
Updated